placement/service/service.go

```diff
@@ -147,16 +137,11 @@ func (ps placementService) AddInstances(
 		addingInstances[i] = addingInstance
 	}
 
-	if ps.opts.Dryrun() {
-		ps.logger.Info("this is a dryrun, the operation is not persisted")
-		return p, addingInstances, err
-	}
-
-	return p, addingInstances, ps.s.CheckAndSet(p, v)
+	return p, addingInstances, ps.Storage.CheckAndSet(p, v)
```

Comment:
Same, you don't need to explicitly reference `Storage` to call its methods unless the method is overriden in the placement service.

Reply:
sg